docs/dev_guide/contents/tests.rst

```diff
@@ -87,6 +87,15 @@ or::
 
     $ pytest --remote-data=any
 
+
+Mocked remote data tests
+------------------------
+Some of our remote data tests are run using `vcrpy <https://vcrpy.readthedocs.io/en/latest/>`_, which saves a copy of any HTTP requests made within the test so a remote request doesn't have to be run on subsequent test runs.
```

Comment:
```suggestion
Some of our remote data tests are run using `vcrpy <https://vcrpy.readthedocs.io/en/latest/>`__, which saves a copy of any HTTP requests made within the test so a remote request doesn't have to be run on subsequent test runs.
```

Reply:
Link seems to work fine with one underscore?

Reply:
It does but if I recall RST convention for links in the middle of sentences should have a double underscore.

Reply:
It's generally a good idea for external URLs to use double underscore.

IIRC the difference between a single and a double is how rst handles repeats of links to the same target. If you link to the same external URL twice when using a single underscore it can cause issues.